ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -22,9 +22,40 @@ parameters:
 - name: BASE_DOMAIN
 - name: CLUSTER_NETWORK_MANIFEST
 - name: ENABLE_FIPS
+- name: IMAGE_EGRESS_HTTP_PROXY
+  required: true
+  value: registry.svc.ci.openshift.org/origin/4.2:egress-http-proxy
```

Comment:
these templates are  used across versions.. ie the same template runs test for 4.1, 4.2, 4.3 etc..

hardcoding to 4.2 therefore seems odd.

Reply:
As far as the cluster-under-test is concerned, this is an opaque proxy.  The OpenShift minor version is orthogonal.  That said, I'd have no problem replacing this pullspec with something floating with `egress-http-proxy`'s latest, so we didn't have to remember to bump this before we drop 4.2 support (in the far, far distant future).  But we haven't been able to find a latest pullspec for the egress image.  Any ideas?